crates/solvers/src/boundary/legacy.rs

```diff
@@ -172,9 +172,7 @@ fn to_boundary_auction(
                 cost: gas.gp_order_cost(),
                 is_liquidity_order: order.class == order::Class::Liquidity,
                 is_mature: true,
-                // Auctions for a /quote request don't have an id and always contain exactly a
-                // single user order that is mandatory to be matched.
-                mandatory: auction.id.is_none(),
+                mandatory: false,
```

Comment:
OK, the purpose of the PR is to make it so there is no real difference between auctions and quotes.